### PR TITLE
Feature/location component

### PR DESCRIPTION
**Pull request by ReenaOmana**
PR for Location Component

### CODE REVIEW

File: core/src/main/java/com/hcl/ecomm/core/config/MagentoServiceConfig.java [hunk 1]
```diff
@@ -22,6 +22,7 @@
   String servicePathCartUpdateItems = "/V1/guest-carts/";
   String guestCartShippingInformationPath = "/us/V1/guest-carts/{cartId}/shipping-information";
   String guestCartCreateOrderPath = "/us/V1/guest-carts/{cartId}/order";
+  String pincodeLocationPath="/content/dam/hclecomm/en/json/state+district+city.json";
```
Comment (rishabh-behl) [inline core/src/main/java/com/hcl/ecomm/core/config/MagentoServiceConfig.java]: @ReenaOmana : dont include opertors in json file name ; pl rename to state_district_city.json

File: core/src/main/java/com/hcl/ecomm/core/config/MagentoServiceConfig.java [hunk 2]
```diff
@@ -128,5 +129,8 @@
   @AttributeDefinition(name = "Customer Get Order Service Path", description = "This is the API path for creating order for customer", defaultValue = customerGetOrders, type = AttributeType.STRING )
   String customer_getOrder_string() default customerGetOrders;
 
+  @AttributeDefinition(name = "Pincode location", description = "This is the Dam path for pincode for state, district and city", defaultValue = {pincodeLocationPath} )
```
Comment (rishabh-behl) [inline core/src/main/java/com/hcl/ecomm/core/config/MagentoServiceConfig.java:132]: @ReenaOmana : pl have a default valu in runmode as well

**Review by rishabh-behl — COMMENTED** [review]
@ReenaOmana : pl check comments and resolve conflicts

File: core/src/main/java/com/hcl/ecomm/core/config/MagentoServiceConfig.java (hunk 1 above)
Comment (ReenaOmana) [reply core/src/main/java/com/hcl/ecomm/core/config/MagentoServiceConfig.java]: File has been renamed now

File: core/src/main/java/com/hcl/ecomm/core/config/MagentoServiceConfig.java (hunk 2 above)
Comment (ReenaOmana) [reply core/src/main/java/com/hcl/ecomm/core/config/MagentoServiceConfig.java:132]: @rishabh-behl  runmode has been updated. Please check

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/clientlibs/clientlib-base/.content.xml [hunk 3]
```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0"
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:mixinTypes="[]"
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/clientlibs/clientlib-base/.content.xml]: @ReenaOmana : why is jcr:mixinTypes="[]" provided for this PR ?

Comment (ReenaOmana) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/clientlibs/clientlib-base/.content.xml]: @rishabh-behl the client lib is updated now. Not sure how the mixinTypes got added here. Will be careful going forward

**Review by rishabh-behl — APPROVED** [review]
LGTM

**Pull request merged**